torrent-stream/torrent-stream.d.ts

```diff
@@ -0,0 +1,62 @@
+// Type definitions for torrent-stream
+// Project: https://npmjs.com/package/torrent-stream
+// Definitions by: Xavier Stouder <https://github.com/xstoudi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../node/node.d.ts" />
+
+
+declare module "torrent-stream" {
+
+    function s(magnet: string | Buffer, options?: TorrentEngineOptions): TorrentEngine;
+
+    namespace s {
+        // Here
+    }
+
+    export = s;
+}
+interface TorrentEngine {
```

Comment:
http://definitelytyped.org/guides/best-practices.html#ghost-modules
Could you use this pattern?